tests/integration/test_installation.py

```diff
@@ -466,11 +466,13 @@ def test_check_inventory_database_exists(ws, new_installation):
     assert err.value.args[0] == f"Inventory database '{inventory_database}' already exists in another installation"
 
 
-@pytest.mark.skip
 @retried(on=[NotFound], timeout=timedelta(minutes=10))
 def test_table_migration_job(  # pylint: disable=too-many-locals
     ws, new_installation, make_catalog, make_schema, make_table, env_or_skip, make_random, make_dbfs_data_copy
 ):
+    # skip this test if not in nightly test job: TEST_NIGHTLY is missing or is not set to "true"
+    if env_or_skip("TEST_NIGHTLY").lower() != "true":
+        pytest.skip("TEST_NIGHTLY is not true")
```

Comment:
```suggestion
    env_or_skip("TEST_NIGHTLY")
```
This one is enough, actually. You can also check if we're running in debug mode and allow it